app/layout.tsx

```diff
@@ -19,10 +18,15 @@ export default function RootLayout({
   return (
     <html lang="en">
       <head>
-        <ColorSchemeScript />
       </head>
-      <body className={inter.className}>
-         <MantineProvider>{children}</MantineProvider>
```

Comment:
Can you expand on the move of `<MantineProvider>` from `layout.tsx` to the root path's `page.tsx` and `PrelaunchSignUpForm.tsx`? 

From [Mantine docs](https://mantine.dev/theming/mantine-provider/):

> It must be rendered at the root of your application and should be used only once.

Reply:
Yes, when I used [createTheme](https://mantine.dev/theming/theme-object/ ) to handle styling the input elements, it did not like being at layout.tsx. I moved it over to PreluanchSignupForm.tsx originally to use the theme there. However, there is a button on the LandingPageTreeVisualizationPanel that is handled by mantine as well. I guess I should have moved the theme into page.tsx at the root, but I wasn't sure about passing the theme from the root for that single page's input fields.

Reply:
There is probably a better method to handle this, and I would love to know if you have some input.